### PR TITLE
Add documentation for Event API

### PR DESCRIPTION
**Pull request by sgny**
Perhaps we can get this included in the release after all.
```
officialDoc: https://github.com/facebook/react-native/blob/master/Libraries/Types/CoreEventTypes.js
```
because code is documentation 😄

### CODE REVIEW

File: reason-react-native/src/apis/Event.md [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
-wip: true
+officialDoc: https://github.com/facebook/react-native/blob/master/Libraries/Types/CoreEventTypes.js
 ---
 
-```reason
-// see https://github.com/facebook/react-native/blob/master/Libraries/Types/CoreEventTypes.js
+These are the types of objects returned in `Event` callbacks. Types are
+instances of `syntheticEvent('a)` and `responderSyntheticEvent('a)`.
+
+## Prametrised Types
```
Comment (MoOx) [inline reason-react-native/src/apis/Event.md]: Parameterized? (it feels weid to me & I was thinking "Generic", but "parameters" & "parameterized" seems what's used in reasonml docs https://reasonml.github.io/docs/en/more-on-type#type-argument

File: reason-react-native/src/apis/Event.md [hunk 2]
```diff
@@ -1,12 +1,17 @@
 ---
 id: apis/Event
 title: Event
-wip: true
+officialDoc: https://github.com/facebook/react-native/blob/master/Libraries/Types/CoreEventTypes.js
```
Comment (MoOx) [inline reason-react-native/src/apis/Event.md]: 😄

File: reason-react-native/src/apis/Event.md (hunk 1 above)
Comment (sgny) [reply reason-react-native/src/apis/Event.md]: Doesn't feel weird to me, but I had to think about what to call these, so I checked the documentation.

**Comment by sgny** [comment]
fixed the typo `Prametrised`

**Pull request merged**